app/src/main/java/com/android/sample/model/activity/Activity.kt

```diff
@@ -39,83 +39,18 @@ enum class ActivityType {
   SOLO,
 }
 
-val types =
-    listOf(
-        ActivityType.PRO,
-        ActivityType.INDIVIDUAL,
-        ActivityType.SOLO,
-    )
+val types = ActivityType.values().toList()
```

Comment:
Great thought! This way, if new types are introduced, the types val will be automatically updated.